lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -536,6 +538,16 @@ interface GetPaymentRequest {
     Lightning(string payment_hash);
 };
 
+enum PaymentReviewAction {
+    "Approve",
+    "Reject",
```

Comment:
Perhaps "Refund" is more explicitly? To answer your question we can move it directly to refundable and let the cooperative refund run.

Reply:
The refund needs some interaction (setting the Bitcoin fee rate). Maybe calling `refund()` if the user rejects the quote is enough? And have a accept quote endpoint for the success case?

Reply:
Makes sense to me which can simplify indeed and then we don't need the enum.

Reply:
Makes sense. Applied in [97d6ca6](https://github.com/breez/breez-sdk-liquid/pull/615/commits/97d6ca687874c66d58b714960d365c33ed8d694f)